components/admin_console/license_settings/license_settings.jsx

```diff
@@ -505,7 +520,12 @@ export default class LicenseSettings extends React.PureComponent {
     renderE0Content = () => {
         let serverError = '';
         if (this.state.serverError) {
-            serverError = <div className='col-sm-12'><div className='form-group has-error'><label className='control-label'>{this.state.serverError}</label></div></div>;
+            serverError = (<div className='col-sm-12'><div className='form-group has-error'><label className='control-label'>
+                <Markdown
+                    enableFormatting={true}
+                    message={this.state.serverError}
+                />
+            </label></div></div>);
```

Comment:
Possible to format this where the opening and closing tags are on the same line? non-blocking

Reply:
I think it looks cleaner this way.